src/Orckestra.Composer.Website/App_Insights/TelemetryInitializer.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Configuration;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+using static Orckestra.Composer.ActionFilters.AppInsightsActionFilter;
+
+namespace Orckestra.Composer.Website.App_Insights
+{
+    public class TelemetryInitializer : ITelemetryInitializer
+    {
+        private static readonly string _operationPrefix 
+            = $"WFE{(ConfigurationManager.AppSettings["Variation"] ?? "").ToUpperInvariant()}";
+
+        static TelemetryInitializer()
+        {
+            string aiKeyFromAppSettings = ConfigurationManager.AppSettings["InstrumentationKey"];
```

Comment:
As the setting name doesn't have "AppInsights" in it, it may be tricky to figure out the meaning of the value for a user that reads the configuration.

You can use the same setting name as the environment variable name that AppInsights expects by default:
"APPINSIGHTS_INSTRUMENTATIONKEY".